homeassistant/components/sensor/geizhals.py

```diff
@@ -0,0 +1,142 @@
+"""
+Parse prices of a device from geizhals.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.geizhals/
+"""
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.util import Throttle
+from homeassistant.helpers.entity import Entity
+from homeassistant.const import (CONF_DOMAIN, CONF_FRIENDLY_NAME, CONF_NAME)
+
+REQUIREMENTS = ['beautifulsoup4==4.6.0']
+_LOGGER = logging.getLogger(__name__)
+
+CONF_product_id = 'product_id'
```

Comment:
Constants must be upper case ([PEP8](https://www.python.org/dev/peps/pep-0008/)).